html/common/templates/assets/assetsShopView.twig

```diff
@@ -38,7 +38,7 @@
                                 <div class="{% if asset.thumbnail|length > 1 %}slick{% endif %}" style="text-align:center;height:200px;">
                                     {% for thumb in asset.thumbnail %}
                                         <div>
-                                            <a href="/asset.php?id={{ asset.assetTypes_id }}{{ SEARCH["SETTINGS"]["SHOWARCHIVED"] ? '&showArchived' : ''}}">
+                                            <a href="{{ CONFIG.ROOTURL }}/asset.php?id={{ asset.assetTypes_id }}{{ SEARCH["SETTINGS"]["SHOWARCHIVED"] ? '&showArchived' : ''}}">
```

Comment:
This causes a 404 on public sites I'm afraid

Reply:
Resolved in 98f45e00